plugins/tech-radar/src/components/RadarPage.tsx

```diff
@@ -49,8 +49,6 @@ export const RadarPage = ({
   return (
     <Page themeId="tool">
       <Header title={title} subtitle={subtitle}>
```

Comment:
Pretty sure this will have to be closed like `<Header title={title} subtitle={subtitle} />` when there are no children.

Reply:
yep - prettier will complain :)

Reply:
yea woops. need to get my prettier pre-commit hook fixed!